### PR TITLE
Opt-out of being able to override built-in elements.

### PR DESCRIPTION
**Pull request by cramforce**
Upgrades `document-register-element` module to `1.6.0`.

See https://github.com/WebReflection/document-register-element/issues/110 for context.

### CODE REVIEW

File: src/polyfills.js [hunk 1]
```diff
@@ -36,7 +36,13 @@ import {getMode} from './mode';
   effect in importing the module.
 */
 if (!getMode().localDev) {
-  installCustomElements(self, 'auto');
+  // Keep in sync with extensions-impl.js
+  installCustomElements(self, {
+    type: 'auto',
+    // No not check whether it is possible to override built-in elements.
+    // AMP does not need this, and not browser as of July 2017 implements it.
+    noBuiltIn: true,
```
Comment (erwinmombay) [inline src/polyfills.js]: `Do not`, `no browser`

**Comment by cramforce** [comment]
This PR will need more fixes to actually work for ES6 transpilation reasons and the CE V1 API.

**Comment by erwinmombay** [comment]
Is it the Reflect problem again?

**Comment by cramforce** [comment]
Yes. Solution is know, though.

**Comment by aghassemi** [comment]
Thanks for the pull request. Since this PR has been stale for a while, I am closing it. Please feel free to reopen as needed.

**Pull request closed without merging**